Library/Homebrew/test/cmd/install_spec.rb

```diff
@@ -17,6 +17,26 @@
     expect(HOMEBREW_CELLAR/"testball1/0.1/foo/test").not_to be_a_file
   end
 
+  it "does not install formulae with forbidden license" do
```

Comment:
Sorry to waste your time on these but we want to avoid adding more integration tests here (because they are really slow). It may be that there's no valid tests for this.

Reply:
no worries